src/can-observable-array.js

```diff
@@ -43,7 +43,8 @@ class ObservableArray extends MixedInArray {
 
 		mixins.finalizeClass(new.target);
 		mixins.initialize(this, props || {});
-
+		this[metaSymbol].preventSideEffects = 0;
```

Comment:
Also, do you know why this property is not initialized? It looks like it should be here: https://github.com/canjs/can-observable-array/blob/1c50a7d278c90b346c4161855568a7aa04621552/src/proxy-array.js#L258.